go/worker/common/api/api.go

```diff
@@ -7,6 +7,9 @@ import (
 
 // Status is the common runtime worker status.
 type Status struct {
+	// Status is an concise status of the committee node.
+	Status string `json:"status"`
```

Comment:
Probably should make its own type with some constants instead of this being directly a string.

Reply:
Added